packages/beacon-node/test/e2e/api/impl/beacon/state/endpoint.test.ts

```diff
@@ -9,8 +9,6 @@ import {getDevBeaconNode} from "../../../../../utils/node/beacon.js";
 import {BeaconNode} from "../../../../../../src/node/nodejs.js";
 
 describe("beacon state api", function () {
-  this.timeout("30s");
```

Comment:
What's the default timeout vitest has? Don't we need to increase the timeout of this test case?

Reply:
Default timeout for vitest is 5s